app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -34,7 +42,7 @@ class ReportsScreen(context: Context, attrs: AttributeSet) : FrameLayout(context
 
     bindUiToController(
         ui = this,
-        events = screenCreates(),
+        events = events,
         controller = controller,
         screenDestroys = detaches().map { ScreenDestroyed() }
```

Comment:
Can we extract `screenDestroys` to a shared stream, just like we did for events, please?

Reply:
We don't need `screenDestroys` as part of the shared stream. It's only required when binding the controller here.

Reply:
@vinaysshenoy can you confirm this once? If I am not using `ScreenDestroyed` in the controller, and it's only being used in `ReplayUntilScreenIsDestroyed`. Do I still need to make the `screenDestroys` as part of the shared stream? 🤔

Reply:
No longer necessary. We will only need it if screen destroyed events are specifically used for something else.